index.bs

```diff
@@ -4612,9 +4640,9 @@ It takes the following input parameters:
 : |requireResidentKey|
 :: The [=effective resident key requirement for credential creation=], a Boolean value determined by the [=client=].
 : |requireUserPresence|
-:: The constant Boolean value [TRUE].
+:: The constant Boolean value [TRUE] unless <code>|options|.{{CredentialCreationOptions/mediation}}</code> is set to {{CredentialMediationRequirement/conditional}}.
     It is included here as a pseudo-parameter to simplify applying this abstract authenticator model to implementations that may
-    wish to make a [=test of user presence=] optional although WebAuthn does not.
+    wish to make a [=test of user presence=] optional.
```

Comment:
Trimming "although WebAuthn does not" entirely could open the door to authenticator models that do weird things for sake of making UP optional in more scenarios.

I know the addition of "unless" in the line above helps narrow down cases when `requireUserPresence` might be set to `FASLE`. But perhaps we need to get hyper-specific on this line here too because otherwise I think it introduces some ambiguity.

Reply:
I think removing the line entirely makes this most clear with the revised line above.